index.html

```diff
@@ -78,6 +78,43 @@ <h2>Introduction</h2>
     Use Cases</a>.
   </p>
 </section>
+<section>
+  <h2> A note on Feature Detection</h2>
+  <p><i>This section is non-normative.</i></p>
+  <p>
+    Feature detection is an established Web development best practice. Resources on the topic are plentiful on and
+    offline and the purpose of this section is not to discuss it further, but rather to put it in the context of
+    detecting hardware-dependent features.
+  </p>
+  <p>
+    Consider the below feature detection examples:
+  </p>
+  <aside class="example" title="Checking existence of ComputePressureObserver interface">
+    <p>
+      This simple example illustrates how to check whether the User Agent exposes the ComputePressureObserver interface
+    </p>
+    <pre class="js">
+      if (typeof ComputePressureObserver == "function") {
+        // get cpu utilization
```

Comment:
Those comments are confusing... like why not the same comment in all? its like one way of doing it should be used for GPU and another for CPU